src/components/PeopleItem/PeopleItem.tsx

```diff
@@ -0,0 +1,72 @@
+import React, { useContext } from 'react';
+import { Person } from '../../types';
+import { useParams } from 'react-router-dom';
+import { PeopleContext } from '../../store/peopleContext';
+import classNames from 'classnames';
+
+type Props = {
+  person: Person;
+};
+
+export const PeopleItem: React.FC<Props> = ({ person }) => {
+  const { persSlug } = useParams();
+  const currPerson = { ...person };
+  const { people } = useContext(PeopleContext);
+
+  if (person.motherName) {
+    currPerson.mother = people.find(p => p.name === person.motherName);
+  }
+
+  if (person.fatherName) {
+    currPerson.father = people.find(p => p.name === person.fatherName);
+  }
+
+  return (
+    <tr
+      data-cy="person"
+      className={classNames({
+        'has-background-warning': persSlug === currPerson.slug,
+      })}
+    >
+      <td>
+        <a
+          className={classNames({ 'has-text-danger': person.sex === 'f' })}
+          href={`#/people/${currPerson.slug}`}
+        >
+          {currPerson.name}
+        </a>
+      </td>
+
+      <td>{currPerson.sex}</td>
+      <td>{currPerson.born}</td>
+      <td>{currPerson.died}</td>
+
+      <td>
+        {currPerson.mother ? (
+          <a
+            className="has-text-danger"
+            href={`#/people/${currPerson.mother?.slug}`}
```

Comment:
Ensure that `currPerson.mother?.slug` is defined before accessing it. If `slug` is not guaranteed to be present, consider adding a check or default value.